admin/marketplace_admin/commands/schedule.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2017 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+
+import re
+import logging
+import subprocess
+from os import path
+from marketplace_admin.services import api
+from marketplace_admin.services import data
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def init_schedule_parser(subparsers):
+    parser = subparsers.add_parser(
+        'schedule',
+        help='Uses cron to regularly call another mktadm command')
+    parser.add_argument(
+        '-H', '--hourly',
+        action='store_true',
+        help='Call the command every hour')
+    parser.add_argument(
+        '-D', '--daily',
+        action='store_true',
+        help='Call the command every day')
+    parser.add_argument(
+        '-W', '--weekly',
+        action='store_true',
+        help='Call the command once a week')
+    parser.add_argument(
+        '-M', '--monthly',
+        action='store_true',
+        help='Call the command once a month')
+    parser.add_argument(
+        '-r', '--remove',
+        action='store_true',
+        help='Removes the specified command from the schedule')
+    parser.add_argument(
+        'schedule_command',
+        type=str,
+        help='The mktadm command to schedule')
+
+    return parser
+
+
+def do_schedule(opts):
+    command = _get_command(opts)
+    crontab = subprocess.check_output(['crontab', '-l']).decode()
```

Comment:
You could catch the CalledProcessError that is raised here when `crontab -l` is run by a user that doesn't have a crontab set up, and set the crontab variable to the empty string.